### PR TITLE
More options, a named package etc. for LocalThickness

### PR DESCRIPTION
**Pull request by rimadoma**

### CODE REVIEW

**Comment by ctrueden** [comment]
@rimadoma Do you know how to edit previous commits? It is very useful when iterating on pull requests.

Please see [this guide](http://imagej.net/How_to_contribute_to_an_existing_plugin_or_library#Modifying_your_commits). And [this guide](http://imagej.net/Git_topic_branches#Advanced_topic_branch_editing_.28AKA_rebase_on_drugs.29). And [this one from Atlassian](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase-i) and [this one from the Git book](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) and [this one from GitHub](https://help.github.com/articles/about-git-rebase/) and [this other one from GitHub](https://help.github.com/articles/using-git-rebase/).

Then you can force push your changes to update this PR, rather than stacking on commits like "Forgot code from previous commit" etc., or opening a new PR every time—both of which get messy fast.

One other "best practices" comment: check out [how to write a good Git commit message](http://chris.beams.io/posts/git-commit/), and our [SCM style guidelines in general](http://imagej.net/Coding_style#SCM_history).

File: src/main/resources/plugins.config [hunk 1]
```diff
@@ -9,8 +9,9 @@
 
 # Generate the jar file using: "jar cvfM LocalThickness_.jar *"
 
-Analyze>Local Thickness, "Local Thickness (complete process)", Local_Thickness_Driver("run")
-Analyze>Local Thickness, "Geometry to Distance Map", EDT_S1D("run")
-Analyze>Local Thickness, "Distance Map to Distance Ridge", Distance_Ridge("run")
-Analyze>Local Thickness, "Distance Ridge to Local Thickness", Local_Thickness_Parallel("run")
-Analyze>Local Thickness, "Local Thickness to Cleaned-Up Local Thickness", Clean_Up_Local_Thickness("run")
+Plugins>Local Thickness, "Local Thickness (complete process)", net.imagej.localThickness.Local_Thickness_Driver
```
Comment (imagejan) [inline src/main/resources/plugins.config]: The "Analyze" menu wasn't such a bad place for this plugin. Putting one more entry to the "Plugins" menu will clutter this even more: it already contains more than 30 entries in a normal Fiji installation, and likely many more when you activated additional update sites.

I would much prefer to have the plugin menu organized in submenus according to image processing techniques. (I know that this is difficult to achieve: when looking for tools related to image registration, you find them in the `Feature Extraction`, `Registration`, `Transform` submenus, but also in `Landmarks`, for example...)

Rainer Engel has recently started an effort to get things organized, but it seems to have stalled:
http://imagej.1557.x6.nabble.com/OT-tool-categories-tp5014809.html

**Comment by rimadoma** [comment]
Fixed issues mentioned by @ctrueden and @imagejan. Didn't write license text, because I don't know which license will apply.

**Comment by rimadoma** [comment]
Don't close request just yet, there's a bug where the plugin crashes, if you close the image while its processing it.

**Comment by hinerm** [comment]
@rimadoma I think it would still be helpful to try out the guides @ctrueden linked to and edit the commit history. At the least, copy and pasting the bullet point summaries you made on GitHub to git commit messages would be fantastic. If that's not clear let us know.

> Didn't write license text, because I don't know which license will apply.

Just so you know you won't have to write this yourself.. everything is automated via the license-maven plugin. Naively, I would expect the license to be [gpl or bsd](http://imagej.net/Licensing)

**Comment by rimadoma** [comment]
@hinerm Edited commit history further with git rebase

**Comment by ctrueden** [comment]
This is looking really good. I am going to merge this as-is, to not hold things up. But I made some comments which I hope help you in the future.

**Comment by ctrueden** [comment]
Well, it seems this doesn't build. I'm on it...

**Comment by ctrueden** [comment]
I cleaned up a bunch of stuff, and the [build](http://jenkins.imagej.net/job/LocalThickness/) is fixed now.

Next time please try to build on the CLI in addition to only within Eclipse.

Anyway, despite that: great work! :beers:

**Pull request merged**